go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/googleapis/gax-go/v2 v2.7.0
 	github.com/jacobsa/daemonize v0.0.0-20160101105449-e460293e890f
 	github.com/jacobsa/fuse v0.0.0-20230509090321-7263f3a2b474
-	github.com/jacobsa/gcloud v0.0.0-20230425120041-5ed2958cdfee
```

Comment:
Please try to keep changes as minimal as possible. Version updates and code changes should be done in seperate PRs.

Reply:
Pulled out into https://github.com/GoogleCloudPlatform/gcsfuse/pull/1259 .

Reply:
This PR should not be merged until [that PR](https://github.com/GoogleCloudPlatform/gcsfuse/pull/1259) is merged.

Reply:
Done